tests/test_jointcal.py

```diff
@@ -0,0 +1,87 @@
+# See COPYRIGHT file at the top of the source tree.
+import unittest
+import unittest.mock
+
+import lsst.log
+import lsst.utils
+
+import lsst.jointcal
+from lsst.jointcal import MinimizeResult
+import lsst.jointcal.chi2
+import lsst.jointcal.testUtils
+
+
+# for MemoryTestCase
+def setup_module(module):
+    lsst.utils.tests.init()
+
+
+class TestJointcalIterateFit(lsst.utils.tests.TestCase):
+    def setUp(self):
+        struct = lsst.jointcal.testUtils.createTwoFakeCcdImages(100, 100)
+        self.ccdImageList = struct.ccdImageList
+        # so that countStars() returns nonzero results
+        for ccdImage in self.ccdImageList:
+            ccdImage.resetCatalogForFit()
+
+        self.config = lsst.jointcal.jointcal.JointcalConfig()
+        # disable both, so it doesn't configure any refObjLoaders
+        self.config.doAstrometry = False
+        self.config.doPhotometry = False
+        self.jointcal = lsst.jointcal.JointcalTask(config=self.config)
+
+        self.goodChi2 = lsst.jointcal.chi2.Chi2Statistic()
+        self.goodChi2.chi2 = 200.0
```

Comment:
Statistically speaking, shouldn't a "good" reduced chi2 be close to 1, not close to 2?